cartographer/sensor/laser.h

```diff
@@ -38,12 +38,10 @@ struct LaserFan {
   std::vector<uint8> reflectivities;
 };
 
-// Builds a LaserFan from 'proto' and separates any beams with ranges outside
-// the range ['min_range', 'max_range']. Beams beyond 'max_range' are inserted
-// into the 'misses' point cloud with length 'missing_echo_ray_length'. The
-// points in both clouds are stored in scan order.
+// Builds a LaserFan from 'proto' and drops any beams with ranges outside
+// the range ['min_range', 'max_range'].
```

Comment:
Add a comment that this is only to drop invalid data and handling short/long returns and missing beams is expected to be handled later?